### PR TITLE
Beautify PR comment

### PR DESCRIPTION
**Pull request by tomszosz**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -95,23 +96,29 @@ runs:
         presidio -f "${{ inputs.output }}" -d "${{ inputs.configuration-data }}" "${{ inputs.path }}" | tee ./.presidio-output.txt
       shell: bash
 
-    - name: Issues not found message
+    
+    - name: Beautify comment message 🧨
```
Comment (cicdguy) [inline action.yaml]:
```suggestion
    - name: Beautify PR comment 🧨
```

File: action.yaml [hunk 2]
```diff
@@ -95,23 +96,29 @@ runs:
...
+      if: ${{ github.event_name == 'pull_request' && contains(inputs.publish, 'true') &&  contains(inputs.output, 'standard') }}
       run: |
-        if [[ ! -s "./.presidio-output.txt" ]]; then
-          echo "There is nothing from presidio check" | tee -a "./.presidio-output.txt"
-        fi
+        echo """
+        PII check result:
```
Comment (cicdguy) [inline action.yaml]:
```suggestion
        ### PII Check
        
        Presidio found potential PII data in the following files:
```

File: action.yaml [hunk 3]
```diff
@@ -95,23 +96,29 @@ runs:
...
+
+        \`\`\`
+        $(cat ./.presidio-output.txt)
```
Comment (cicdguy) [inline action.yaml]: Handle the case where there are no findings or if the file is empty

File: action.yaml [hunk 4]
```diff
@@ -95,23 +96,29 @@ runs:
...
+        \`\`\`
```
Comment (cicdguy) [inline action.yaml]: Make this whole section collapsible.

File: action.yaml (hunk 3 above)
Comment (tomszosz) [reply action.yaml]: modified

File: action.yaml (hunk 4 above)
Comment (tomszosz) [reply action.yaml]: modified

**Review by cicdguy — APPROVED** [review]

**Comment by cicdguy** [comment]
LGTM

**Pull request merged**